packages/banner/src/Banner.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import Icon from '@leafygreen-ui/icon';
```

Comment:
We can now import just the glyph we want to reduce file size thanks to @DesignerDave's merged PR!

Example of what we can (and should now) do! https://github.com/mongodb/leafygreen-ui/pull/412/files#diff-ab0dc90acb8625429ef9c29cecbee9ee

Reply:
Ah didn't realize this was merged!! That's epic @DesignerDave

Reply:
👍